R/RunDiagnostics.R

```diff
@@ -143,6 +149,8 @@ runCohortDiagnostics <- function(packageName = NULL,
     databaseDescription <- databaseId
   }
   
+  cohortTable <- cohortTableNames$cohortTable
```

Comment:
if we dont do assert check, what if this is not a named list object, or cohortTableNames$cohortTable returns NULL